CONTRIBUTING.md

```diff
@@ -14,26 +14,113 @@ If in doubt about a policy, please [ask on our Slack](https://communityinviter.c
 
 No changes, ***no matter how trivial***, are ever made to the contents of published specifications.  The only potential changes to those documents are updates to link URLs _if and only if_ the targeted document is moved by a 3rd party.  Other changes to link URLs are not allowed.
 
-### Current branches and documents open to change
+### Changing the schemas
 
-The first PR for a change should be against the oldest release line to which it applies.  Changes can then be forward-ported as appropriate.
+Schemas are only changed _after_ the specification is changed.
+Changes are made to the YAML versions on the `main` branch.
+The JSON versions are generated when published to the [spec site](https://spec.openapis.org/), at which time the `WORK-IN-PROGRESS` URI placeholders are replaced with the publication date.
 
-The current (31 July 2024) active releases are:
+### Authoritative source of truth
 
-| Version | Branch | File | Notes |
-| ------- | ------ | ---- | ----- |
-| 3.0.4 | `v3.0.4-dev` | `versions/3.0.4.md` | Soon to be published |
-| 3.1.1 | `v3.1.1-dev` | `versions/3.1.1.md` | Soon to be published |
-| 3.2.0 | `v3.2.0-dev` | `versions/3.2.0.md` | Planned for late 2024 |
-| 4.0.0 | [OAI/sig-moonwalk](https://github.com/OAI/sig-moonwalk) | [discussions only](https://github.com/OAI/sig-moonwalk/discussions) | TBD, some proposals to be backported and published in 3.2.0 |
+The [spec site](https://spec.openapis.org) is the source of truth.
 
-### Changing the schemas
+This changed in 2024, as the markdown files on `main` do not include certain credits and citations.
+
+## Development and publication process
+
+As of October 2024 (post-OAS 3.0.4 and 3.1.1), the OAS is developed in the `src/oas.md` file on minor release `X.Y-dev` branches that are derived from the baseline `dev` branch.
```

Comment:
```suggestion
As of October 2024 (post-OAS 3.0.4 and 3.1.1), the OAS is developed in the `versions/next.md` file on minor release `X.Y-dev` branches that are derived from the baseline `dev` branch.
```
Why not keep it next to the published versions?

Reply:
@ralfhandl because one of our problems is that people confuse the published and under-development files.

We want the development to be clearly separated from the publishing.  These are two different functions.

If all of our documentation talks about editing `src/oas.md`, it's much harder to confuse that with `versions/whatever.md`.  Also, the source file isn't a "versions" in the sense of the published files, so putting it in that directory is a category error in my view.

Honestly, my preference would be to not publish through GitHub at all, and _only_ publish directly to `spec.openapis.org`.  That would also allow us to dispense with the `X.Y.Z-rel` branches because the file name wouldn't matter anymore.  But I figured that would be too big of a change (deleting the existing published files off of `main` would probably upset people).